### PR TITLE
Update the Git download link in the Developement Section for Windows

### PR DESCRIPTION
**Pull request by Ptchopat**
Old
Git - https://git-scm.com/download/
image

New
Git - https://git-scm.com/downloads/win
image

### CODE REVIEW

File: README.md
Comment (gridhead) [inline README.md:437]: Thank you for your contribution.

Could you please make the following change?

We want to include support for both GNU/Linux distributions and Microsoft Windows.

```suggestion
    Git - https://git-scm.com/downloads
```

**Review by gridhead — CHANGES_REQUESTED** [review]

File: README.md
Comment (Ptchopat) [reply README.md:448]: Great idea.
I added the appropriate links and some useful informations for the installation in GNU/Linux distributions and Microsoft Windows.

**Review by gridhead — CHANGES_REQUESTED** [review]
Good progress so far.

Could you please squash the changes to just one commit?

**Comment by sdglitched** [comment]
You need not include the changes in your branch which are not related to it. Also, no need to update the branch as `git` will take care of it.

Please squash the changes to just one commit as advised by @gridhead

**Comment by Ank-22** [comment]
@gridhead, What's the hold up with this merge?

**Comment by gridhead** [comment]
@Ank-22 The contributor has not responded yet. Please feel free to take up the task in the meanwhile.

**Comment by gridhead** [comment]
Closing without merge as the maintainer has not responded since a couple of weeks now. 

Superseding via #194

**Pull request closed without merging**
